apps/web/components/email-list/EmailList.tsx

```
@@ -364,6 +365,71 @@ export function EmailList({
+  //         onArchive(thread);
+  //       }
+  //     }
+  //   }
+
+  //   window.addEventListener("keydown", handleKeyDown);
+  //   return () => window.removeEventListener("keydown", handleKeyDown);
+  // }, [threads, focusedIndex, setOpenThreadId, onArchive, refetch]);
+
+  // const [focusedIndex, setFocusedIndex] = useState(0);
+
+  const handleAction = useCallback(
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: onArchive

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)